script.js

```diff
@@ -260,6 +284,62 @@ function handleLocalitiesLayer(geojson) {
   return localitiesLayer;
 }
 
+function handleCountiesLayer(geojson) {
+  // styling for the localities layer: style localities conditionally according to a presence of a moratorium
+  const pointToLayer = (feature, latlng) => {
+    // style localities based on whether their moratorium has passed
+    if (feature.properties.passed === "Yes") {
+      return L.circleMarker(latlng, {
+        color: "#4dac26",
+        fillColor: "#b8e186",
+        fillOpacity: fillOpacity / 3,
+        radius: pointRadius*1.5,
+        weight: strokeWeight*2
+      });
+    } else {
+      return L.circleMarker(latlng, {
+        color: "#d01c8b",
+        fillColor: "#f1b6da",
+        fillOpacity: fillOpacity/3,
+        radius: pointRadius*1.5,
+        weight: strokeWeight*2
+      });
+    }
+  };
+
+  // Create the Leaflet layer for the localities data
+  const countiesLayer = L.geoJson(geojson, {
+    pointToLayer: pointToLayer
+  });
+
+  // Add popups to the layer
+  countiesLayer.bindPopup(function(layer) {
+    // This function is called whenever a feature on the layer is clicked
+
+    // Render the template with all of the properties. Mustache ignores properties
+    // that aren't used in the template, so this is fine.
+    const renderedInfo = Mustache.render(
+      infowindowTemplate,
+      layer.feature.properties
+    );
+    document.getElementById(
+      "aemp-infowindow-container"
+    ).innerHTML = renderedInfo;
+    return Mustache.render(popupTemplate, layer.feature.properties);
+  });
+
+  // Add data to the map
+  countiesLayer.addTo(map);
+
+  // Move the map view so that the localitiesLayer is visible
+  map.fitBounds(countiesLayer.getBounds(), {
```

Comment:
@goldfarb there's also a similar piece of code in the `handleLocalities` function. We probably don't want `map.fitBounds` being called multiple times after the data has loaded, doing it on just one layer is probably sufficient.